assets/src/components/NavigationType/NavigationType.js

```diff
@@ -0,0 +1,31 @@
+import { RadioControl } from '@wordpress/components';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+
+const NAVIGATION_TYPE_PLANET4 = 'planet4';
+const NAVIGATION_TYPE_MINIMAL = 'minimal';
+
+/**
+ * Navigation type selector
+ */
+export const NavigationType = ({value, setValue, defaultValue, options}) => {
+  const defaultOptions = [
+    {
+      "label": __( "Main website navigation", 'planet4-blocks-backend' ),
```

Comment:
Small detail, but why do you use a mix o quotes? I mean `''` and `'`.

Reply:
Probably some copy/pasting thing :grimacing: